packages/cloudflare/README.md

```diff
@@ -161,6 +161,15 @@ The following example automatically generates `_routes.json` while including and
   });
 ```
 
+### `imageService`
+
+`imageService: "passthrough" | "cloudflare"`
+
+Determines which image service is used by the adapter. The following options are available, if no option is provided the adapter automaticly overrides to passtrough mode if an uncompatible image service is used, otherwise it just uses the configured image service:
```

Comment:
```suggestion
Determines which image service is used by the adapter. The following options are available, if no option is provided the adapter will default to  `passthrough ` mode when an incompatible image service is configured. Otherwise, it will use the globally configured image service:
```

Reply:
Yeah that is better than my version, but I think it is still quite hard to understand the default option, but that is not very straightforward technically.

Reply:
Fair enough 😁, either way try to fix the typos in among those lines: 
- automaticly => automatically
- uncompatible => incompatible

Reply:
We can wait for someone else to jump in and tweak it even more 😁